ui/src/pageLayout/containers/Nav.test.tsx

```diff
@@ -2,29 +2,38 @@
 import React from 'react'
 import {render} from 'react-testing-library'
 import {IconFont} from 'src/clockface'
+import _ from 'lodash'
 
 // Components
 import NavMenu from 'src/pageLayout/components/NavMenu'
 
+// Constants
+const DASHBOARDS_NAV_ITEM = 'Dashboards'
+const TASKS_NAV_ITEM = 'Tasks'
+
 function Nav(props) {
   const {pathname} = props
+
+  const activateNavItem = (keywords: string[]): boolean => {
```

Comment:
it seems weird that this function is being defined in the test as well as the component. should it be moved to a util to be reused. Additionally, should this test include the exact implementation of determining what should be highlighted?

Reply:
That's a good call